### DISCUSSION TITLE
Facing issues in OTA with 64kB bin file

### DISCUSSION THREAD

**Discussion by talhayousuff** [JAndrassy/ArduinoOTA#93] [Q&A]
Hi jandrassy, I have been able to perform OTA on a 1.5kB bin file with ATMEGA2560 and SIM800C. However, if I try it on a 64kB bin file, the same code is unable to download the complete file and resets during downloading. Any guidance in this regard would be helpful.

**Comment by JAndrassy** [comment]
yes, it is a known error of the ArduinoHttpClient library
https://github.com/arduino-libraries/ArduinoHttpClient/pull/103

**Reply by talhayousuff** [reply]
Thank you for responding, I have updated the library but I am still having the same issue.

**Reply by JAndrassy** [reply]
the library is not fixed
